libfswatch/src/libfswatch/c/windows/realpath.c

```diff
@@ -21,9 +21,9 @@
 #include <io.h>
 #include <stdlib.h>
 #include <errno.h>
+#include "cfswatch_export.h"
```

Comment:
This file is not generated by the build system, so the build breaks when using the Autotools